target/linux/mvebu/image/cortex-a9.mk

```diff
@@ -59,7 +64,8 @@ endef
 TARGET_DEVICES += linksys_wrt32x
 
 define Device/linksys_wrt1900ac
-  $(call Device/linksys,WRT1900AC (Mamba))
+  $(call Device/linksys)
+  DEVICE_MODEL := WRT1900AC (Mamba)
```

Comment:
Okay, this is the V1 for the V2 above. So we got us another V1 discussion here.

Reply:
;-)